texastribune/donations
# Waco glowup ancillary pages

#### What's this PR do?
Updates the charge_waco and error pages to have Waco-specific details/design.

#### Why are we doing this? How does it help us?
Waco time

#### How should this be manually tested?
* Set the NEWSROOM Vals (_NAME, _TITLE, _DOMAIN) to be Waco-specific
* spin up the app
* attempt giving a donation and check out how the charge page looks afterwards
* visit /error and check out how the error page looks

#### How should this change be communicated to end users?
It's a part of the BIGGER WACO PICTURE

#### Are there any smells or added technical debt to note?
Uhh, yeah. In general this app is moving closer to being more dynamic but this in-between state has definitely got the app in its "awkward teenager" years. Some clean-up/rethinking/redesigning is needed to reach a more optimal way of handling different site uses. But hey, it's working.

#### What are the relevant tickets?
https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwS1XPty68eK4Ett/recfcd0ARMPOhMvfY?blocks=hide

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
